### PR TITLE
[Book] Remove pandas requirement

### PR DESCRIPTION
**Pull request by Herklos**

### CODE REVIEW

File: octobot_trading/data/book.py [hunk 1]
```diff
@@ -13,42 +13,161 @@
...
-    def get_bids(self, side=TradeOrderSide.BUY.value):
-        return self.orders.query(f"side.str.contains('{side}')", engine='python').values.tolist()
+def _order_id_index(order_id, order_list):
+    """
+    Return order id index in order list
+    :param order_id: the order id to search
+    :param order_list: the order list to check
+    :return: the order id index in order list if found else ORDER_ID_NOT_FOUND
+    """
+    for i in range(order_list):
```
Comment (GuillaumeDSM) [inline octobot_trading/data/book.py:170]: Not sure this works, I guess it's one of the reasons for WIP :)

File: tests/data/test_book.py [hunk 2]
```diff
@@ -0,0 +1,157 @@
+#  Drakkar-Software OctoBot-Trading
```
Comment (GuillaumeDSM) [inline tests/data/test_book.py:1]: 💯

File: octobot_trading/data/book.py (hunk 1 above)
Comment (Herklos) [reply octobot_trading/data/book.py:170]: Yes :D

File: octobot_trading/data/book.py [hunk 3]
```diff
@@ -13,42 +13,157 @@
...
-        return self.orders.query(f"side.str.contains('{side}')", engine='python').values.tolist()
+def _order_id_index(order_id, order_list):
+    """
+    Return order id index in order list
+    :param order_id: the order id to search
+    :param order_list: the order list to check
+    :return: the order id index in order list if found else ORDER_ID_NOT_FOUND
+    """
+    if order_list is not None:
+        for i in range(len(order_list)):
```
Comment (GuillaumeDSM) [inline octobot_trading/data/book.py:166]: should we use enumerate(order_list) ?

Comment (Herklos) [reply octobot_trading/data/book.py:166]: Yes it's better with enumerate :+1:

**Pull request merged**